web/controllers/admin_controller.ex

```diff
@@ -319,6 +319,7 @@ defmodule ExAdmin.AdminController do
   end
 
   defp to_integer(:string, string), do: string
+  defp to_integer(:id, string), do: string
```

Comment:
I presume you are working in either master or themes. Probably best if you work in themes. I will be merging pull request into my themes branch first, then I'll merge into master.  With that said...

Under test, you should see a `schema.exs`, `migrations.exs`, ... in `test/support`. You can add any schema and migrations into these files. Then you can add any admin stuff into `test/support/admin_resources.exs`. Finally, you can update one of the existing test files or add a new one. In your case, I would say add a test to `test/controller_test.exs`.

Does that answer your question?